src/onegov/ticket/model.py

```diff
@@ -107,9 +116,10 @@ def created(cls):
         'title': {'type': 'text'},
         'subtitle': {'type': 'text'},
         'group': {'type': 'text'},
-        'ticket_email': {'type': 'keyword'},
-        'ticket_data': {'type': 'localized_html'},
-        'extra_localized_text': {'type': 'localized'}
+        # properties cannot be added to search index
+        # 'ticket_email': {'type': 'keyword'},
+        # 'ticket_data': {'type': 'localized_html'},
+        # 'extra_localized_text': {'type': 'localized'}
```

Comment:
Don't modify the search properties. You can make them work with tsvector by changing them to hybrid_properties. The function that generates the tsvector should probably emit an warning if it encounters a property name which it cannot translate to a tsvector statement. In the future it should be a hard failure, but we definitely don't want to change the search results for the existing elastic search.

Reply:
hybrid_property support added while generating tsvector string and re-enabled all properties in Ticket